packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -71,6 +71,11 @@ import 'bottom_tab_bar.dart';
 /// )
 /// ```
 ///
+/// To push a route above all tabs instead of inside the currently selected one
+/// (such as when showing a dialog on top of this scaffold), use
+/// `Navigator.of(rootNavigator: true)` from inside the [BuildContext] of a
```

Comment:
I know this is just for documentation, so this PR handles that, but this is probably a poor coding practice.  It suggests that there is something special about 2 level navigation when in reality, we could have any number of levels of navigation.  We should consider some kind of ID system for Navigators, instead.

Reply:
Nested navigators are all individually addressable via `GlobalKey<NavigatorState>`